nautobot_netbox_importer/diffsync/models/abstract.py

```diff
@@ -198,6 +201,21 @@ def create_nautobot_record(nautobot_model, ids: Mapping, attrs: Mapping, multiva
             record = nautobot_model(**ids, **attrs)
             record.clean()
             record.save()
+
+            # To keep track of original created date of a ChangeLoggedModel that is updated
+            # when `record.save()` and to create a ChangeObject with the import change
+            if issubclass(type(record), ChangeLoggedModel):
+                record.created = model_data["created"]
+
+                ObjectChange.objects.create(
+                    changed_object=record,
+                    object_repr=str(record),
+                    action="update",
+                    object_data=serialize_object(record),
+                    # # Random request_id as it's mandatory
```

Comment:
@glennmatthews maybe we could add a well-know fake UUID to signal it better? other ideas?

Reply:
How much does it mess up the UI if you use the same UUID for *all* ObjectChange created in a single run of the importer? I know that's not typical under normal usage.

Reply:
I don't think it would be a problem, I will check it out